i18n/en/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,47 @@ sidebar_position: 4
 
 # Naming
 
-## Problem
+Different developers have different experiences and contexts, which can lead to misunderstandings on the team when the same entities are called differently. For example:
 
-Due to the fact that each developer has his own experience and development context - [we are used to calling the same entities differently][disc-src], which can lead to misunderstandings within the team.
+- Components for display can be called "ui", "components", "ui-kit", "views", …
+- The code that is reused throughout the application can be called "core", "shared", "app", …
+- Business logic code can be called "store", "model", "state", …
 
-- *Components to be displayed - `ui` / `components` / `ui-kit` / `views` / ...*
-- *Code reusable in all parts of the application - the `core`/ `shared` / `app` / ...*
-- *Code business logic `store` / `model` / `state` / ...*
+## Naming in Feature-Sliced Design {#naming-in-fsd}
 
-## Naming in FSD
+The methodology uses specific terms such as:
 
-The methodology uses such terms as
+- "app", "process", "page", "feature", "entity", "shared" as layer names,
+- "ui', "model", "lib", "api", "config" as segment names.
 
-- `app`, `process`, `page`, `feature`, `entity`, `shared` - *[layers][refs-layers]*
-- `ui`, `model`, `lib`, `api` - *[segments][refs-segments]*
+Original naming is crucial to prevent confusion among team members and new developers joining the project. Using unique names also helps when seeking help from the community.
```

Comment:
issue: I think the meaning has slightly changed here from the previous version, and I think that the previous version was more on point here:

* Before: when developing a project, it is important to adhere to the original naming
* After: original naming is important

The difference here is subtle, to me, the Before version conveys "don't change the naming" and the After version conveys "use naming that hasn't been used before".

suggestion:
```suggestion
It is crucial to preserve these terms to prevent confusion among team members and new developers joining the project. Using standard names also helps when seeking help from the community.
```

Reply:
Fixed.